events/iisc_pulse_events.txt

```diff
@@ -128,23 +143,32 @@ event = {
 					days = 1
 				}
 			}
+			country_event = {
+				id = iisc_pulse.20
+				days = 30
+			}
 		}
 	}
 }
 
 # build ships if necessary
 # checks if enough research to get a tech
-# once per month
+# once per 30 days
 # triggered by iisc_pulse.20
+# hq set
 country_event = {
 	id = iisc_pulse.21
 	is_triggered_only = yes
 	hide_window = yes
 	trigger = {
-		exists = overlord
+		exists = event_target:iisc_hq_planet
+		NOR = {
+			has_global_flag = iisc_stopped
+			has_global_flag = iisc_paused
+		}
```

Comment:
Where is `iisc_stopped` added? How is it distinct from `iisc_paused`?

Reply:
Nowhere for now. This will need more PR ^^
The logic I thought of:
paused => momentarily paused (no corps spawn, less computations, ...)
stopped => prepare for desintall (kill the corps, their ships, ... stop the recurring events)